### PR TITLE
consul 1.6.5

### PR DESCRIPTION
**Pull request by i0rek**

### CODE REVIEW

**Comment by i0rek** [comment]
After this is merged, I will remove the old versions.

File: library/consul [hunk 1]
```diff
@@ -6,10 +6,10 @@ GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: 9cb67358af03c73222da3badebc7c69d765c456a
 Directory: 0.X
 
-Tags: 1.6.4, 1.6
+Tags: 1.6.5, 1.6
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
-GitCommit: 59bdf0ae8863e3166aadc81e18f8781a52a07136
+GitFetch: refs/heads/consul-1.6.5
```
Comment (yosifkit) [inline library/consul:12]: This still needs a `GitCommit` along with the `GitFetch` so that it knows which commit on that branch.

Comment (i0rek) [reply library/consul:12]: Thank you for your help!

**Comment by tianon** [comment]
LGTM

Build test of #7976; 117c43f2; `amd64` (`consul:1.6.5`):

```console

$ bashbrew build consul:1.6.5
Building bashbrew/cache:761017d2f6355c6bdb0133f07973391c0d1ebf27d96e1d47b57b286c6bf271c3 (consul:1.6.5)
Tagging consul:1.6.5
Tagging consul:1.6

$ test/run.sh consul:1.6.5
testing consul:1.6.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

(The build failure / flakiness exhibited in Actions could probably be improved by adding `gpgconf --kill all` somewhere before the `rm`, if you felt so inclined.)

**Pull request merged**